### PR TITLE
fix: split user and id token due to size, add auth0 example

### PR DESCRIPTION
**Pull request by jensneuse**

### CODE REVIEW

File: examples/auth0-oidc-authentication/example.env [hunk 1]
```diff
@@ -0,0 +1 @@
+FAUNADB_TOKEN="Bearer <replace-with-your-token>"
```
Comment (StarpTech) [inline examples/auth0-oidc-authentication/example.env:1]: delete

File: examples/auth0-oidc-authentication/next.config.js [hunk 2]
```diff
@@ -0,0 +1,5 @@
+module.exports = {
```
Comment (StarpTech) [inline examples/auth0-oidc-authentication/next.config.js:1]: I think this file can be deleted as well.

File: examples/auth0-oidc-authentication/package.json [hunk 3]
```diff
@@ -0,0 +1,32 @@
+{
+  "name": "wundergraph-inject-bearer",
```
Comment (StarpTech) [inline examples/auth0-oidc-authentication/package.json:2]: rename to `wundergraph-oidc`

File: examples/auth0-oidc-authentication/pages/index.tsx [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+	c7-10.1,11.1-22.2,11.1-35.4c0-34.4-27.9-62.2-62.2-62.2s-62.2,27.9-62.2,62.2s27.9,62.2,62.2,62.2c1.8,0,3.5-0.1,5.3-0.3l52.2,90.3
+	c-24.9,42.7-39,92.6-39,145.4c0,80.1,32.4,152.6,84.9,205.1c52.5,52.5,125,84.9,205.1,84.9c151,0,275.4-115.7,288.7-263.5
+	c0.8-8.8,1.3-17.5,1.3-26.5v-26.5H675.4z M553.4,733.2c-64.5,0-122.8-26.3-165-68.4c-42.2-42.2-68.5-100.6-68.5-165
+	c0-30.5,5.8-59.7,16.7-86.5L484.4,669l69-116.7l68.5,116.5l83.8-142.5H785C772,642.8,673.3,733.2,553.4,733.2z"
+							/>
+						</svg>
+					</div>
+				</div>
+				<h1 className="text-slate-900 font-extrabold text-4xl sm:text-5xl lg:text-6xl tracking-tight text-center dark:text-white">
+					FaunaDB & Next.js
```
Comment (StarpTech) [inline examples/auth0-oidc-authentication/pages/index.tsx:42]: Update title

File: pkg/authentication/authentication.go [hunk 5]
```diff
@@ -707,7 +734,18 @@ func (u *UserLogoutHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 		SameSite: http.SameSiteStrictMode,
 		Secure:   !u.InsecureCookies,
 	}
-	http.SetCookie(w, c)
+	http.SetCookie(w, userCookie)
```
Comment (StarpTech) [inline pkg/authentication/authentication.go:737]: Could you move the "reset" cookie logic into a function?

**Review by StarpTech — CHANGES_REQUESTED** [review]

File: examples/auth0-oidc-authentication/package.json [hunk 6]
```diff
@@ -0,0 +1,32 @@
+{
+  "name": "wundergraph-auth0-oidc",
+  "version": "0.1.0",
+  "private": true,
+  "scripts": {
+    "start": "run-p dev wundergraph open",
+    "wundergraph": "cd .wundergraph && wunderctl up -e ./../.env --debug",
+    "generate": "cd .wundergraph && wunderctl generate -e ./../.env --debug",
+    "open": "wait-on -d 500 http://localhost:9991 && open-cli http://localhost:3003",
```
Comment (StarpTech) [inline examples/auth0-oidc-authentication/package.json:9]: All demos starting with 3000? should we slign?

Comment (jensneuse) [reply examples/auth0-oidc-authentication/package.json:9]: I'd actually give every example an incremented port so you can run multiple demos concurrently.

**Review by StarpTech — APPROVED** [review]

**Pull request merged**